feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewUiState.kt

```diff
@@ -40,7 +40,8 @@ sealed interface PreviewUiState {
         val toastMessageToShow: ToastMessage? = null,
         val snackBarToShow: SnackbarData? = null,
         val lastBlinkTimeStamp: Long = 0,
-        val previewMode: PreviewMode
+        val previewMode: PreviewMode,
+        val captureModeToggleUiState: CaptureModeToggleUiState
```

Comment:
nit: Set this to default to `CaptureModeToggleUiState.Invisible` so you don't need to add it as an argument to all of the compose Preview functions.

Reply:
I don't think this is true. If the previewMode is ExternalImageCapture, we'd still have to set this to Visible in PreviewViewModel's initialization.

Reply:
I'm in favor of not setting a default value to make us think about the toggle state each time PreviewUiState is made since the logic for displaying/hiding has a lot of corner cases, and the fact that it's going to need another condition added to it when externalVideoCapture is implemented